src/puya/__main__.py

```diff
@@ -88,6 +88,41 @@ def main() -> None:
         default=False,
         help="Output MIR before lowering to TealOps",
     )
+    parser.add_argument(
+        "--output-bytecode",
+        action=argparse.BooleanOptionalAction,
+        default=False,
+        help="Output AVM bytecode",
+    )
+    parser.add_argument(
+        "--match-algod-bytecode",
+        action=argparse.BooleanOptionalAction,
+        default=False,
+        help="When outputting bytecode, ensure bytecode matches algod output",
+    )
+    parser.add_argument(
+        "-T",
+        "--template-var",
+        dest="cli_template_definitions",
+        metavar="VAR=VALUE",
+        action="append",
+        help="Define template vars for use when assembling via --output-bytecode"
+        " should be specified without the prefix (see --template-vars-prefix), e.g."
+        " -T SOME_INT=1234"
+        " -T SOME_BYTES=0x1A2B"
+        ' -T SOME_STR=\\"hello\\"',
+    )
+    parser.add_argument(
```

Comment:
thoughts on having a switch to enable using environment variables instead? could be more flexible for e.g. CI applications?

Reply:
After further discussion, decided it's safer and simpler to just have the CLI option and remove the option to source variables from a file